src/main/java/org/example/Main.java

```diff
@@ -1,22 +1,31 @@
 package org.example;
 import java.util.Scanner;
 
-import static java.lang.System.in;
-import static java.lang.System.out;
+import static java.lang.System.*;
 
 public class Main {
 
     public static final String CORRECT_NUMBER = "5";
+    public static  int MAX_TRIES = 5;
+    public static int NUMBER_TRIES= 0;
 
     public static void main(String[] args) {
-        Scanner number = new Scanner(in);
-        out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
-        String name = number.nextLine();
-        out.println(name);
-        if (CORRECT_NUMBER.equals(name)) {
-            out.println("Wszedles");
-        } else {
-            out.println("Nie prawidlowe haslo");
+        while (NUMBER_TRIES < MAX_TRIES) {
+            Scanner number = new Scanner(in);
+            out.println("Wpisz " + CORRECT_NUMBER + " lub inny numer");
+            String name = number.nextLine();
+            out.println(name);
+            if (CORRECT_NUMBER.equals(name)) {
+                out.println("Wszedles");
+                break;
+            } else {
+                NUMBER_TRIES++;
+                out.println("Nie prawidlowe haslo");
+            }
+        }
+        if (NUMBER_TRIES == MAX_TRIES)//dupa123
```

Comment:
don't commit irrelevant commits, especially profanities
Also, this whole condition doesn't make sense - when you get to the end of while loop, you don't have to check anything, you already know you exceeded number of tries